templates/webapi/test/overview.html.ep

```diff
@@ -5,6 +5,7 @@
 % use List::Util qw(sum);
 
 % content_for 'ready_function' => begin
+  window.overviewOkResultSelector = '<%= $ok_result_sel %>';
```

Comment:
this looks to be inconsistent. In other places you say "…collapsable_results_sel" but here "ok_result_sel" even though it's configurable so not necessarily the same as the real definition of "ok" results that we use elsewhere

Reply:
My thinking was the selector defines what "ok" results are in that particular occasion. However, it is likely better to stick with "collapsable_results_sel".